berkeley-mobile/Common/AlertView.swift

```diff
@@ -0,0 +1,190 @@
+            blurEffectView.frame = self.view.bounds
+            
+            blurEffectView.alpha = 0.75
+            blurEffectView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+
+            view.addSubview(blurEffectView)
+            blurEffectView.translatesAutoresizingMaskIntoConstraints = false
+            
+            blurEffectView.topAnchor.constraint(equalTo: view.topAnchor).isActive = true
+            blurEffectView.leftAnchor.constraint(equalTo: view.leftAnchor).isActive = true
+            blurEffectView.rightAnchor.constraint(equalTo: view.rightAnchor).isActive = true
+            blurEffectView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
+            
+            view.sendSubviewToBack(blurEffectView)
+            self.tabBarController?.tabBar.addSubview(blurEffectView)
+
+        } else {
+            view.backgroundColor = .black
+        }
+        
+    }
+    
+    
+    @objc func btn1Function() {
+        btn1Action()
+        NotificationCenter.default.post(name: NSNotification.Name(rawValue: "PopupDidAppear"), object: nil)
+    }
+    
+    @objc func btn2Function() {
+        btn2Action()
+        NotificationCenter.default.post(name: NSNotification.Name(rawValue: "PopupDidAppear"), object: nil)
+    }
+    
+    @objc func dismissPop() {
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        setupView()
+    }
+    
+    
+    func setupView(){
+        
+        let backgroundView = UIView()
+        backgroundView.backgroundColor = .white
+        backgroundView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(backgroundView)
+        
+        let Vstack = UIStackView()
+        Vstack.axis = .vertical
+        Vstack.spacing = 10
+        Vstack.distribution = .fillProportionally
+        
+        
+        Vstack.addArrangedSubview(heading)
+        Vstack.addArrangedSubview(message)
+        Vstack.translatesAutoresizingMaskIntoConstraints = false
+        
+
+        
+        let Hstack = UIStackView()
+        Hstack.axis = .horizontal
+        Hstack.distribution = .fillEqually
+        Hstack.spacing = 15
+        
+        Hstack.addArrangedSubview(btn1)
+        if !withOneButton {
+            Hstack.addArrangedSubview(btn2)
+        }
+        Hstack.translatesAutoresizingMaskIntoConstraints = false
+        
+        backgroundView.addSubview(Vstack)
+        backgroundView.addSubview(Hstack)
+        
+        
+        backgroundView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        backgroundView.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+        backgroundView.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 304/375).isActive = true
```

Comment:
nitpick: reduce whitespace, we typically don't want to have 2 lines of whitespace in a row